RxAlamofire/RxAlamofireTests/RxAlamofireTests.swift

```diff
@@ -69,4 +71,40 @@ class RxAlamofireSpec: XCTestCase {
             XCTFail("\(error)")
         }
 	}
+
+    func testProgress() {
+        do {
+            let dataRequest = try request(HTTPMethod.get, "http://myjsondata.com").toBlocking().first()!
+            let progressObservable = dataRequest.rx.progress().observeOn(MainScheduler.instance).replayAll()
```

Comment:
I don't think you need `.observeOn(MainScheduler.instance)` here. You are hijacking dataRequest, so the only events that will be replayed are progress events you are sending lower in the method :) Interesting way of doing this :)